crates/services/tx_status_manager/src/config.rs

```diff
@@ -8,6 +9,8 @@ pub struct Config {
     pub subscription_ttl: Duration,
     /// Maximum time to keep the status in the cache of the manager.
     pub status_cache_ttl: Duration,
+    /// Protocol Signing Key, i.e. the block signer's public key
+    pub protocol_public_key: PublicKey,
```

Comment:
I think this is a left over. If not it's not needed check to see how I implemented it without using it : https://github.com/FuelLabs/fuel-core/pull/2848

Reply:
Removed.